### DISCUSSION TITLE
Ability to move images or add empty lines between images

### DISCUSSION THREAD

**Discussion by iliapir2** [outline/outline#1320] [General]
Hello outline team! Thank you for your great work! It's feels and looks brilliant.

I have one idea on how we can make outline even better. 

As I understand now it's impossible to move uploaded images in the document or add empty lines between them. My colleagues stuck in the situation when they want to add some text between images and can't do it. See screenshot below:

I can do a PR to fix this behavior, please point me in the right direction

![](https://habrastorage.org/webt/pg/rd/dn/pgrddnhdwrxzeeqthvzcoku_itk.png)

**Comment by tommoor** [comment]
Agreed, there are a couple of separate little issues here, one of which is filed:

- The second spot you point to is the image caption, you should be able to place the cursor in there and hit enter to get a new paragraph – issue here: https://github.com/outline/rich-markdown-editor/issues/226

- The first is actually the gap between between the title and the start of the editor, are you able to place the cursor in the title and press the enter key to get a new line? That might also be an improvement to make.

**Reply by iliapir2** [reply]
> The second spot you point to is the image caption, you should be able to place the cursor in there and hit enter to get a new paragraph – issue here: outline/rich-markdown-editor#226

Would like to help you with this. Leave a comment on that thread

> The first is actually the gap between between the title and the start of the editor, are you able to place the cursor in the title and press the enter key to get a new line? That might also be an improvement to make.

Nope, I'm not able to get a newline before the image. Should I create a separate Feature Request?

**Reply by tommoor** [reply]
> Should I create a separate Feature Request?

Sure thing, that issue would be in this repo – as the title is part of Outline, not the editor.
